### DISCUSSION TITLE
How to Run the Project After Cloning the Repository Locally

### DISCUSSION THREAD

**Discussion by ezekel** [manchenkoff/nuxt-auth-sanctum#225] [Q&A] [answered]
Hi @manchenkoff 

Can I ask a question? I'm cloning the repo to my local machine and want to play with it. Can you help me figure out how to run it locally? Should I just run npm install followed by npm run dev? Also, if I add another folder to the src directory with a simple script or function, how do I call that function in the playground folder and access it from the Vue pages?

Thank you in advance!

**Comment by manchenkoff** [comment, marked as answer]
Hey @ezekel! Sure thing, you can find all the details about the environment in `CONTRIBUTING.md`. If you want to start a playground then `pnpm dev` should be enough.

> Also, if I add another folder to the src directory with a simple script or function, how do I call that function in the playground folder and access it from the Vue pages?

For instance, if you want to add a new composable function, it will be available in `#imports` after the build and you can import it in your component or page. You can check `pages/login.vue` as an example of the `useSanctumAuth` import. The same will be true for `utils` and other directories that are supported by Nuxt. However, if you want to create a random file in a different folder, it might not work, so I would suggest keeping it directly in the `playground` scope.
